[EDU-5296] Neon Database with Drizzle template guide (#1334)
aziontech/docs: src/content/docs/en/pages/guides/marketplace/templates/neon-database-with-drizzle.mdx

---
title: How to deploy the Neon Database Starter Kit with Drizzle ORM template
description: This template contains the configurations to build and connect an application to Neon Database.
meta_tags: Drizzle ORM, Neon Database, Edge Functions, Azion Marketplace, web, guides, templates
namespace: docs_guides_neon_database_with_drizzle
permalink: /documentation/products/guides/neon-database-with-drizzle/
---

import Tag from 'primevue/tag';

<Tag severity="info" client:only="vue">
Preview
</Tag>

The **Neon Database Starter Kit with Drizzle ORM** template contains the configurations for creating and connecting an edge application to a Neon database via Drizzle ORM.

The deployment of this template creates a GitHub repository containing your project, as well as an edge application and a domain to facilitate your access and management through Azion Edge Platform. It also creates a new table in your Neon database.

---

## Requirements

- A [GitHub account](https://github.com/signup) to connect with Azion and create your new repository.
    - Every push will be deployed automatically to this repository to keep your project updated.

> - A [Neon database](https://neon.tech/) with space available for a new table.

Reviewer: ```suggestion
- A [Neon database](https://neon.tech/) with available space for a new table.
```

> This template uses [Application Accelerator](/en/documentation/products/build/edge-application/application-accelerator/), [Edge Functions](/en/documentation/products/build/edge-application/edge-functions/), and [Edge Cache](https://www.azion.com/en/documentation/products/build/edge-application/edge-cache/), and it could generate usage-related costs. Check the [pricing page](https://www.azion.com/en/pricing/) for more information. Database storage and operations could also generate costs directly by Neon. Check [Neon's pricing page](https://neon.tech/pricing) for details.

Reviewer: ```suggestion
This template uses [Application Accelerator](/en/documentation/products/build/edge-application/application-accelerator/), [Edge Functions](/en/documentation/products/build/edge-application/edge-functions/), and [Edge Cache](/en/documentation/products/build/edge-application/edge-cache/), and it could generate usage-related costs. Check the [pricing page](https://www.azion.com/en/pricing/) for more information. Database storage and operations could also generate costs directly by Neon. Check [Neon's pricing page](https://neon.tech/pricing) for details.
```

> This template uses [Application Accelerator](/en/documentation/products/build/edge-application/application-accelerator/), [Edge Functions](/en/documentation/products/build/edge-application/edge-functions/), and [Edge Cache](https://www.azion.com/en/documentation/products/build/edge-application/edge-cache/), and it could generate usage-related costs. Check the [pricing page](https://www.azion.com/en/pricing/) for more information. Database storage and operations could also generate costs directly by Turso. Check [Turso's pricing page](https://turso.tech/pricing) for details.

Author: ```suggestion
This template uses [Application Accelerator](/en/documentation/products/build/edge-application/application-accelerator/), [Edge Functions](/en/documentation/products/build/edge-application/edge-functions/), and [Edge Cache](https://www.azion.com/en/documentation/products/build/edge-application/edge-cache/), and it could generate usage-related costs. Check the [pricing page](https://www.azion.com/en/pricing/) for more information. Database storage and operations could also generate costs directly by Neon. Check [Neon's pricing page](https://neon.tech/pricing) for details.
```
